lldb/tools/lldb-dap/DAP.h

```diff
@@ -60,7 +60,7 @@
 
 namespace lldb_dap {
 
-typedef llvm::DenseMap<std::pair<uint32_t, uint32_t>, SourceBreakpoint>
+typedef std::map<std::pair<uint32_t, uint32_t>, SourceBreakpoint>
```

Comment:
changed to `std::map`  because I'm not sure how to remove from the map while iterating it with `DenseMap`